app/lib/project/MetapolatorProject.js

```diff
@@ -614,6 +616,38 @@ define([
         exportController.export();
     };
 
+    _p.exportZipInstance = function(masterName, instanceName, precision) {
+        //export the font to a temporary directory
+        var temp_dir = instanceName+"_temp";
+        this.exportInstance(masterName, temp_dir, precision);
```

Comment:
We would still need to eigther rmDir the subtree that is stored in memory or delete the InMemory object to free the used browser memory. Destroing the object seems easier, but I don't see code for that in place at InMemory.js

Will it be freed automatically by the javascript VM via some sort of garbage collecting mechanism? Or do we need to explicitely destroy the inMemory object before getting rid of our last reference to it?